cclib/parser/molcasparser.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of cclib (http://cclib.github.io), a library for parsing
+# and interpreting the results of computational chemistry packages.
+#
+# Copyright (C) 2014-2015, the cclib development team
+#
+# The library is free software, distributed under the terms of
+# the GNU Lesser General Public version 2.1 or later. You should have
+# received a copy of the license along with cclib. You can also access
+# the full license online at http://www.gnu.org/copyleft/lgpl.html.
```

Comment:
It looks like you put the wrong license here. It should reference the BSD 3-clause license.

Reply:
Thanks!

Reply:
Yes, please use the same header as the other parsers: https://github.com/cclib/cclib/blob/master/cclib/parser/gaussianparser.py#L1